cmd/soroban-cli/src/commands/tx/args.rs

```diff
@@ -0,0 +1,92 @@
+use soroban_rpc::GetTransactionResponse;
+
+use crate::{
+    commands::{global, txn_result::TxnResult},
+    config::{self, address, data, network, secret},
+    fee,
+    rpc::{self, Client},
+    tx::builder,
+};
+
+#[derive(Debug, clap::Args, Clone)]
+#[group(skip)]
+pub struct Args {
+    #[clap(flatten)]
+    pub fee: fee::Args,
+    #[clap(flatten)]
+    pub config: config::Args,
+    //// The source account for the operation, Public key or Muxxed Account
+    /// e.g. `GA3D5...` or `MA3D5...`
+    #[arg(
+        long,
+        visible_alias = "with_source",
+        env = "STELLAR_WITH_SOURCE_ACCOUNT"
+    )]
+    pub with_source_account: Option<address::Address>,
```

Comment:
Can we make this option named the same as the option is named in other commands? i.e. `--source-account` with alias `--source`?

Reply:
This is the source account for the operation. Should it be renamed to `op_source_account`/`op_source`?

Reply:
Ah. Given these commands are a convenience for single operation transactions, I don't think they need to support setting the operation source. There's few cases where in a single op transaction that a user plans to submit right then that they need a different source account as on the transaction. The only reason would be to disconnect the sequence number, which is a valid thing to need to do, but I suspect rare in this use case.

We can add it later if we need to.